graspy/pipeline/mug2vec.py

```diff
@@ -0,0 +1,161 @@
+            edges would have received. Number of possible edges is determined 
+            by the type of graph (loopless or looped, directed or undirected).
+        - None
+            No pass to ranks applied. 
+
+    omnibus_components, cmds_components : int or None, default = None
+        Desired dimensionality of output data. If "full", 
+        n_components must be <= min(X.shape). Otherwise, n_components must be
+        < min(X.shape). If None, then optimal dimensions will be chosen by
+        ``select_dimension`` using ``n_elbows`` argument.
+
+    omnibus_n_elbows, cmds_n_elbows: int, optional, default: 2
+        If `n_components=None`, then compute the optimal embedding dimension using
+        `select_dimension`. Otherwise, ignored.
+    
+    Attributes
+    ----------
+    omnibus_n_components_ : int
+        Equals the parameter n_components. If input n_components was None,
+        then equals the optimal embedding dimension.
+
+    cmds_n_components_ : int
+        Equals the parameter n_components. If input n_components was None,
+        then equals the optimal embedding dimension.
+
+    embeddings_ : array, shape (n_components, n_features)
+        Principal axes in feature space from classical MDS.
+
+    See also
+    --------
+    graspy.utils.pass_to_ranks
+    graspy.embed.OmnibusEmbed
+    graspy.embed.ClassicalMDS
+    graspy.embed.select_dimension
+    """
+
+    def __init__(
+        self,
+        pass_to_ranks="simple-nonzero",
+        omnibus_components=None,
+        omnibus_n_elbows=2,
+        cmds_components=None,
+        cmds_n_elbows=2,
+    ):
+        self.pass_to_ranks = pass_to_ranks
+        self.omnibus_components = omnibus_components
+        self.omnibus_n_elbows = omnibus_n_elbows
+        self.cmds_components = cmds_components
+        self.cmds_n_elbows = cmds_n_elbows
+
+    def fit(self, graphs, y=None):
+        """
+        Computes a vector for each graph.
+
+        Parameters
+        ----------
+        graphs : list of nx.Graph or ndarray, or ndarray
+            If list of nx.Graph, each Graph must contain same number of nodes.
+            If list of ndarray, each array must have shape (n_vertices, n_vertices).
+            If ndarray, then array must have shape (n_graphs, n_vertices, n_vertices).
+        
+        y : Ignored
+
+        Returns
+        -------
+        self : returns an instance of self.
+        """
+        if pass_to_ranks is not None:
+            graphs = [pass_to_ranks(g, self.pass_to_ranks) for g in graphs]
+
+        omni = OmnibusEmbed(
+            n_components=self.omnibus_components, n_elbows=self.omnibus_n_elbows
+        )
+        omnibus_embedding = omni.fit_transform(graphs)
+
+        self.omnibus_n_components_ = omnibus_embedding.shape[-1]
+
+        cmds = ClassicalMDS(
+            n_components=self.cmds_components, n_elbows=self.cmds_n_elbows
+        )
+        self.components_ = ClassicalMDS().fit_transform(omnibus_embedding)
```

Comment:
is this supposed to be `embeddings`? And not using the `cmds` object created above